templates/default/rt/rtticketinfoshort.html

```diff
@@ -21,6 +21,16 @@
 		</TD>
 	</TR>
 	{/if}
+	{if $ticket.createtime}
+	<TR>
+		<TD class="bold nobr">
+			<i class="lms-ui-icon-uptime"></i> {trans("Creation time:")}
```

Comment:
To już proponuję trzymać się zasady:
```smarty
{icon name="uptime"} {trans("Creation time")}
```
Nie lepiej `Create time` zamiast `Creation time`?

Reply:
Może warto do tego dorobić parametr "label" ?
np. zamiast `{icon name="uptime"} {trans("Creation time")}` 

mogłoby być:
{icon name="uptime" label="Creation time:"}

??